src/components/Application/ProgressBar.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Accordion, Container, Grid, Header, Label, Sticky } from 'semantic-ui-react'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { TemplateSectionPayload } from '../../utils/types'
+
+interface SectionPage {
+  sectionIndex: number
+  currentPage: number
+}
+
+interface ProgressBarProps {
+  serialNumber: string
+  templateSections: TemplateSectionPayload[]
+  sectionPage?: SectionPage
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  serialNumber,
+  templateSections,
+  sectionPage,
+}) => {
+  const childrenPanels = (step: number, totalPages: number) => {
+    const { push } = useRouter()
+    const pages = Array.from(Array(totalPages).keys(), (n) => n + 1)
+    return pages.map((number) => {
+      return {
+        key: `progress_${step}_${number}`,
+        title: {
+          children: (
+            <Header
```

Comment:
I know we're not really worrying about styles right now, but structurally wouldn't this perhaps be better as `<li>` elements? 

Or in Semantic, it's the `<List><ListItem>...` components.

Reply:
Ah, but then we loose the `Accordion` functionality. I tried a few different things for this side progress bar, and although it still seems a little funky I still think the best element here would be an accordion. Let me know if I not properly understood your suggestion here.

Reply:
I just meant to use `<li>` or Semantic `<List>` instead of the `<Header>` component here. That should still work with accordion, shouldn't it?

I'm not actually _seeing_ "Accordion" functionality (as in a collapsing toggle arrow). Should I be?

Reply:
> I'm not actually seeing "Accordion" functionality (as in a collapsing toggle arrow). Should I be?

I have replaced the toggle arrow with the numbers to conform with what was the UI design for these.

> I just meant to use <li> or Semantic <List> instead of the <Header> component here.

I'll try to update this now. : )